packages/x-data-grid/src/components/panel/filterPanel/filterPanelUtils.ts

```diff
@@ -31,3 +35,31 @@ export function getValueFromValueOptions(
   });
   return getOptionValue(result);
 }
+
+/**
+ * Find the option matching the given value in valueOptions and get its label
+ * @param {string} value is used to extract label from valueOptions.
+ * @param {any[] | undeinfed} valueOptions is used to extract label.
```

Comment:
we have a type for this here: https://github.com/mui/mui-x/blob/1613993896045cbb7fecbbc848066143368c284b/packages/x-data-grid/src/models/colDef/gridColDef.ts#L25

Reply:
oh, and there is a type
```suggestion
 * @param {any[] | undefined} valueOptions is used to extract label.
```